src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,29 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+
+    private static LocalDate localDate = LocalDate.now();
```

Comment:
let's move this field to all methods where it is needed

Reply:
done